back/engines/commercial/bulk_import_ideas/app/policies/bulk_import_ideas/patches/project_policy.rb

```diff
@@ -8,7 +8,7 @@ def bulk_create_async?
       end
 
       def export_form?
-        active_moderator?
+        true
```

Comment:
Made sense that the form should be available to anyone as then it can be linked to from anywhere

Reply:
Yes, just changed it to mirror `show?`